app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,70 @@
+*{
```

Comment:
this will be applied everywhere not only on the login page, if it is what you want to do, than move it to bioportal.scss

Reply:
Ok